src/homeWork_001/Diamond.java

```diff
@@ -0,0 +1,27 @@
+package homeWork_001;
+
+public class Diamond {
```

Comment:
create separate method, for example drawDiamond(int sizeOfRhombus). And if you write sizeOfRhombus = 3, you must get next picture: first line - one asterisk; second line - three asterisks; third line - one asterisk.